cmd/history.go

```diff
@@ -37,42 +37,34 @@ func newHistoryCmd() *cobra.Command {
 		SuggestFor: []string{"updates"},
 		Short:      "Update history for a stack",
 		Long: "Update history for a stack\n" +
```

Comment:
If we don't do this for other commands then we should leave this as-is, but it might be cleaner to use a multi-line string literal here. That might clean up the code ever so slightly. e.g.:

```go
            Long: `Update the history for a stack

This command lists data about previous updates for a stack.`,
            Args: cmdutil.NoArgs,
```